Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/Core.gwt.xml

```diff
@@ -41,6 +41,11 @@ knowledge of the CeCILL-B license and that you accept its terms.
     <inherits name="com.google.gwt.i18n.I18N" />
     <inherits name='com.google.gwt.user.theme.standard.Standard' />
 
+    <inherits name="com.google.gwt.logging.Logging"/>
+    <set-property name="gwt.logging.enabled" value="TRUE" />
+    <set-property name="gwt.logging.logLevel" value="FINEST" />
+    <set-property name="gwt.logging.consoleHandler" value="ENABLED" />
```

Comment:
Isn't it redondent with the same Logging module inherited in Application.gwt.xml ? I think it should only be in Core.gwt.xml.